### PR TITLE
Multi-instance

### PR DESCRIPTION
**Pull request by carsten19**
- Add support for multi-instance associations, requires openzwave-shared 1.5.8
- Fix display of helper messages (did not work for some time apparently)
- Remove dead code from admin page (referring to commands array which don't even exist)
- Add materialize admin
- Performance improvements on start (use AssumeAwake flag and don't call extendNode twice)
- Tooltips for network map to explain the color
- Bump version

### CODE REVIEW

File: admin/index.html [hunk 1]
```diff
@@ -563,37 +560,41 @@
...
                                     var id = 'zwave.' + instance + '.NODE' + $('#assNodeIds').val() + '.sleep';
-                                    if (values[id] && !values[id].val) {
+                                    if (values[id] && values[id].val) {
                                         // don't refresh when the node is asleep as getAssociations is not updated
+                                        showMessage(_('Your selection will be saved once the device wakes up'), _('Important'), 'alert');
+                                    } else {
                                         setTimeout(function(){
                                             $('#assGroups').val(1).selectmenu('refresh').val(currentGroup).selectmenu('refresh')
-                                        }, 1000);
+                                        }, 1000);
```
Comment (AlCalzone) [inline admin/index.html]: Trailiing whitespace
grafik

File: main.js [hunk 2]
```diff
@@ -1167,7 +1205,9 @@ function main() {
 
         // Just remember, that such a nodeID created
         nodes[nodeID] = {id: calcName(nodeID), ready: false, native: null};
-        extendNode(nodeID, {});
+        if (inclusion) {
+            extendNode(nodeID, {});
```
Comment (AlCalzone) [inline main.js:1209]: Are you sure about this? Doesn't that mean the objects are not re-created if they were deleted within ioBroker?

File: package.json [hunk 3]
```diff
@@ -50,7 +50,8 @@
     "chai": "^4.1.2",
     "gulp": "^4.0.0",
     "mocha": "^6.0.2",
-    "axios": "^0.18.0"
+    "axios": "^0.18.0",
+    "materialize-css": "^1.0.0"
```
Comment (AlCalzone) [inline package.json]: Afaik, that is already included in ioBroker admin - no need to embed it here

File: README.md [hunk 4]
```diff
@@ -80,7 +80,6 @@ If all of those commands don't work, the following process might:
 ## Configuration
 Within Admin Settings you can set following Attributes
 
-- Force objects re-init (ReInitialize all Objects within ioBroker)
```
Comment (AlCalzone) [inline README.md:83]: Whats the reason for this?

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
In the future, please split changes into separate PRs with a clear intent. Makes it way easier to review than this. I just had a quick glance but I'm pretty sure I missed something.

File: main.js (hunk 2 above)
Comment (carsten19) [reply main.js:1209]: The node added event is called during inclusion (which is still covered), during startup (where it is superfluous as it just overwrites existing data just to revert it again in "node available") and during refresh where it is still covered by "node available".

File: README.md (hunk 4 above)
Comment (carsten19) [reply README.md:83]: That function doesn't exist. Maybe it did in the past.

File: main.js (hunk 2 above)
Comment (AlCalzone) [reply main.js:1209]: Ok so we still create the node objects if they are missing in the `node available` event?

Comment (carsten19) [reply main.js:1209]: Yes that is correct

**Comment by carsten19** [comment]
Changes committed, done from my side

**Review by AlCalzone — APPROVED** [review]
LGTM

**Pull request merged**